src/controller/Controls.jsx

```diff
@@ -8,7 +8,6 @@ const Controls = ({
 }) => (
   <>
```

Comment:
You should wrap this all in a form with an onSubmit instead of using a fragment!